runtime/v2/shim/util_unix.go

```diff
@@ -63,20 +66,20 @@ func AdjustOOMScore(pid int) error {
 	return nil
 }
 
-// SocketAddress returns an abstract socket address
+// SocketAddress returns a socket address
 func SocketAddress(ctx context.Context, id string) (string, error) {
```

Comment:
Should add some warning in release note: 

It will be failed to add new container grouped into existing shimv2.
For example, containerd shutdown for upgrading but the CRI-plugin only creates pause sandbox. When containerd restarts, CRI-plugin will fail to create container in pod because of changed socket path.

Reply:
That is a tricky one.  I think, with a little more work, I can work it to support existing abstract sockets based on the static `/containerd-shim` path (outside of `/run`).  We can just assume this is an abstract socket path.  Does that sound ok to you?

Reply:
I think from @AkihiroSuda comment above about prefixing the paths with `file://` or `unix://` then we can solve this problem this way.  If there is not URI based path in the `address` file, we will treat it as an abstract socket for backwards compatibility.  I think this will work out really well :)

Reply:
Sure. It sounds good to me. Thanks!